test/00-simple.bats

```diff
@@ -55,3 +55,55 @@ load helpers
 	run podman run --net=host --security-opt seccomp=${tmpFile} alpine ls
 	[ "$status" -eq 0 ]
 }
+
+@test "Containers fails to run blocked syscall"{
+	local tmpFile
+	local size
+
+	tmpFile=$(mktemp)
+	echo "Temporary file: ${tmpFile}"
+
+	run podman run --net=host --annotation io.containers.trace-syscall=of:${tmpFile} ${ALPINE} ls
+	echo "Podman output: ${lines[*]}"
+	[ "$status" -eq 0 ]
+	# sleep two seconds to let the hook finish writing the file
+	sleep 2
+
+	size=$(du -b ${tmpFile} | awk '{ print $1 }')
+	echo "Size of generated file: ${size}"
+	[ "${size}" -gt 0 ]
+
+	run podman run --net=host --security-opt seccomp=${tmpFile} alpine sync
+	[ "$status" -ne 0 ]
+}
+
+@test "Extend existing seccomp profile"{
+	local tmpFile1
+	local tmpFile2
+	local size 
+
+	tmpFile1=$(mktemp)
+	tmpFile2=$(mktemp)
+	echo "Temporary file 1: ${tmpFile1}"
+	echo "Temporary file 2: ${tmpFile2}"
+
+	run podman run --net=host --annotation io.containers.trace-syscall=of:${tmpFile1} ${ALPINE} ls
+	[ "$status" -eq 0 ]
+	# sleep two seconds to let the hook finish writing the file
+	sleep 2
+
+	size=$(du -b ${tmpFile1} | awk '{ print $1 }')
+	echo "Size of the first generated file: ${size}"
+	[ "${size}" -gt 0 ]
+
+	run podman run --net=host --annotation io.containers.trace-syscall="if:${tmpFile1};of:${tmpFile2}" ${ALPINE} echo "Alpine"
+	[ "$status" -eq 0 ]
+	sleep 2
+
+	size=$(du -b ${tmpFile2} | awk '{ print $1 }')
+	echo "Size of the second generated file: ${size}"
+	[ "${size}" -gt 0 ]
+
+	run podman run --net=host --security-opt seccomp=${tmpFile2} alpine echo "Alpine"
```

Comment:
`echo` seems to be a subset of `ls`. Let's use `ping -c3 google.com` to trigger a different set of syscalls.